examples/jakstat_signaling/simulate_jakstat_pesto.m

```diff
@@ -0,0 +1,243 @@
+% simulate_jakstat_pesto.m is the matlab interface to the cvodes mex
```

Comment:
I dont think this file makes sense if we do not ship compiled mex simulation routines.